pkg/server/router/did.go

```diff
@@ -222,6 +222,52 @@ type ResolveDIDResponse struct {
 	DIDDocumentMetadata *didsdk.DIDDocumentMetadata   `json:"didDocumentMetadata,omitempty"`
 }
 
+type DeleteDIDByMethodRequest struct {
+	// Identifies the cryptographic algorithm family to use when generating this key.
+	// One of the following: `"Ed25519","X25519","secp256k1","P-224","P-256","P-384","P-521","RSA"`.
+	KeyType crypto.KeyType `json:"keyType" validate:"required"`
+
+	// Required when creating a DID with the `web` did method. E.g. `did:web:identity.foundation`.
+	DIDWebID string `json:"didWebId"`
```

Comment:
should this be required too?

Reply:
If we had it required then when you would make a did-key you would have to have didWebId set to blank (or anything)

Definitely should change this, would require some refactoring on how we handle the did creation endpoint

Reply:
gotcha

I think we could amend this to be a generic "option" field that has unique meaning per DID method

Reply:
Added a new issue - https://github.com/TBD54566975/ssi-service/issues/335